extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -96,6 +96,7 @@ export let InteractiveReactData;
  *    canShowSharingUis: boolean,
  *    canShowSystemLayerButtons: boolean,
  *    customControls: !Array<!Object>,
+ *    playerHasNextStoryState: boolean,
```

Comment:
As seen offline, let's get the Player to send a new configuration that can add/remove/enable/disable buttons.